Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -5352,8 +5352,15 @@ private void DoGetRecordings(bool learnAdditionalChanges)
                                     {
                                         configArgs.LearnedElementInfo = eInfo;
                                     }
+                                    else
+                                    {
+                                        //if the element info object is not found then adding action by existing method not supporting POM
```

Comment:
i didn't meant that in case eInfo is null so you will add the action directly to BF, we still need to work with POM so i meant that if it is null so you will create the elementInfo object here with whatever info you know to add to it like locateBy, locateValue, elementType... and then return that to RecordingManager to add it to POM and create action out of it- any issue with achiving that?

Reply:
Done